src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -83,7 +85,12 @@ const CreateInstance = (props: CreateInstanceProps): JSX.Element => {
         >
           {t("instance.createSEInstance")}
         </Button>,
-        <Button key="cancel" variant="link" onClick={onClose}>
+        <Button
+          key="cancel"
+          ouiaId="cancel-instance-modal"
```

Comment:
Here we might need just `cancel` as ouiaId.
The fact that it belongs to a modal is inferred from surrounding context. `Modal[@ouiaId=create-instance] > Button[@ouiaId=cancel]`